kubeflow/hack/delete_gcp.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
-set -e
+set -ex
 
-echo "Deleting all GCP resources will cause destruction of all services and data on this cluster. Confirm? [y/N]"; 
-read REPLY; 
-if [[ $REPLY =~ ^[Yy]$ ]]
-then 
-    kubectl --context=$MGMTCTXT delete -f common/cnrm/$BUILD_DIR
+if [[ -z "${MGMTCTXT}" ]]; then
+    echo "MGMTCTXT env var is required"
+    exit 1
+fi
+
+echo "Deleting all GCP resources will cause destruction of all services and data on this cluster. Confirm? [y/N]";
```

Comment:
NIT:`... will cause destruction of all services and data except Cloud SQL instance and GCS bucket. ```